docs/src/language_servers.md

```diff
@@ -542,6 +542,33 @@ There are multiple options:
     }
     ```
 
+### Steep
+
+1. Add the `steep` gem (see [github:soutaro/steep](https://github.com/soutaro/steep)) into your Gemfile and install it
+
+    ```sh
+    bundle install
+    ```
+
+2. Binstub steep executable
+
+    ```sh
+    steep binstub
+    ```
+
+3. Open `Preferences > Package Settings > LSP > Settings` and add the `"steep"` client configuration to the `"clients"`:
+
+    ```jsonc
+    {
+        "clients": {
+            "steep": {
+                "command": ["bin/steep", "langserver"],
+                "selector": "source.ruby | text.html.ruby",
+            }
+        }
+    }
+    ```
+
```

Comment:
Did you forgot to include the fourth step?
```suggestion

4. Activate server for the currect project - open Command Palette `LSP: Enable Language Server in Project > steep`
```

Reply:
Ok, I saw this commit now [75a2468](https://github.com/sublimelsp/LSP/pull/2377/commits/75a246881a64051aac3332f2cac62613efff8b70)

Reply:
It is true that the instruction for others do not have this step,
but they do have:
```
"enabled": true,
```